tests/ops/integration_tests/saas/test_hubspot_task.py

```diff
@@ -12,7 +13,9 @@
 from fidesops.ops.task.filter_results import filter_data_categories
 from fidesops.ops.task.graph_task import get_cached_data_for_erasures
 from fidesops.ops.util.saas_util import format_body
+from tests.ops.fixtures.saas.hubspot_fixtures import user_exists
 from tests.ops.graph.graph_test_util import assert_rows_match, records_matching_fields
+from tests.ops.test_helpers.saas_test_utils import poll_for_existence
 
 
```

Comment:
Let's add a connection test so we can tell more easily if we're running into an auth issue when the tests start failing.

```
@pytest.mark.integration_saas
@pytest.mark.integration_hubspot
def test_hubspot_connection_test(hubspot_connection_config) -> None:
    get_connector(hubspot_connection_config).test_connection()
```

Reply:
done!!!